src/cljs/org/broadinstitute/firecloud_ui/page/workspaces_list.cljs

```diff
@@ -121,9 +136,9 @@
            :header [:div {:style {:marginLeft -6}} "Status"] :starting-width 60
            :content-renderer (fn [data] [StatusCell {:data data}])}
           {:as-text :name :sort-by :text
-           :header "Workspace" :starting-width 450
+           :header "Workspace" :starting-width  (* max-workspace-name-length 10)
            :content-renderer (fn [data] [WorkspaceCell {:data data}])}
-          {:header "Description" :starting-width 400
+          {:header "Description" :starting-width (* max-description-length 10)
            :content-renderer (fn [description]
                                [:div {:style {:padding "1.1em 0 0 14px"
                                               :fontStyle (when-not description "oblique")}}
```

Comment:
is there any way to integrate the "No description required" here with the one in get-max-workspace-description-length? Ideally, we wouldn't have that hardcoded string in both places.